src/components/ReactTable/ReactTable.tsx

```diff
@@ -167,6 +169,12 @@ export const ReactTable = <T extends {}>({
     usePagination,
   )
 
+  useEffect(() => {
+    if (!onPageChange) return
```

Comment:
Didn't find a way to do this with `useTable` FWIW, it's a controlled component (well, hook) and doesn't expose such event handlers

Reply:
I couldn't find any ways on my side too, looks fine for me